react/src/components/molecules/Link/index.js

```diff
@@ -6,12 +6,13 @@ import { Icon } from '../../../index';
 
 const Link = (props) => {
   const icon = props.icon ? (<Icon name={props.icon} svgWidth={13.2} svgHeight={13.2} />) : '';
+  const classes = (props.classes) ? `js-clickable-link ${props.classes}` : 'js-clickable-link';
```

Comment:
```suggestion
```

Reply:
js-clickable-link is not used

Reply:
I don't think this should be done. There's no defaultProp set for classes, so just assuming it always exists when it's not required is bad.

Reply:
I think having an unused class there is also not a great solution. can we set it to null? or conditionally passing className?